editor/index.html

```diff
@@ -222,6 +222,9 @@
 
 			}
 
+			window.editor = editor;
+			window.THREE = THREE;
```

Comment:
If you want to expose `THREE` at this place, you have to check whether the following line is still necessary or not:

https://github.com/mrdoob/three.js/blob/6c8c7e75f0f04888a814fbebd395cb2055151711/editor/js/libs/app.js#L9

Reply:
If we need `editor` exposed only for scripts, it might be better to move `window.editor = editor;` to `app.js`.

Reply:
Ah thanks for the heads up. With this change that line is indeed no longer necessary.

I don't think editor is needed for scripts, though. I think it's best to remove `window.THREE` from this PR and move `window.editor` to line 70 of `index.html`.

Reply:
> With this change that line is indeed no longer necessary.

Um, removing the line in `app.js` might break published projects since they are standalone applications. Meaning they do not run within the editor's context.

Reply:
Ah sorry, I didn't mean to say we should remove the line from `app.js`. I think we can leave `app.js` as it is. I just meant to say that putting `window.THREE` in the project twice is unnecessary. So we can remove either the one from this PR or the one from `app.js`. I think it's best to remove it from this PR.

Reply:
Well, then feel free to do so 😇

Reply:
Done :)